big_tests/tests/mam_SUITE.erl

```diff
@@ -994,6 +1014,26 @@ required_modules(muc_no_elements, Config) ->
     Opts = #{muc := MUC} = ?config(mam_meta_opts, Config),
     NewOpts = Opts#{muc := MUC#{no_stanzaid_element => true}},
     [{mod_mam, NewOpts}];
+required_modules(muc_light_failed_to_decode_message_in_database, Config) ->
+    Opts = #{muc := MUC} = ?config(mam_meta_opts, Config),
+    NewOpts = Opts#{muc := MUC#{db_message_format => mam_message_eterm}},
+    [{mod_mam, NewOpts}];
+required_modules(muc_light_failed_to_decode_message_in_database2, Config) ->
```

Comment:
No, please don't do this. It's super confusing to find a non-existing test case name here. Maybe add a helper like this:

```
required_modules(MessageType, MessageFormat) ->
  Opts = #{MessageType := Cfg} = ?config(mam_meta_opts, Config),
  NewOpts = Opts#{MessageType := Cfg#{db_message_format => MessageFormat}},
```

This helper could be then used in the test case.